try {
    helloWorld();
} catch (error) {
    console.log(error);
}


try {
    helloWorld();
} catch {
    console.log('Error');
}